### ISSUE TITLE
Switching Camera Issue

### ISSUE THREAD

**Issue by hayrer** [slavavdovichenko/MediaLibDemos3x#18]
the application crashes then i toggle camera from Back to Front in Portrait DeviceOrientation
![screen shot 2015-02-06 at 13 02 07](https://cloud.githubusercontent.com/assets/1833198/6078374/8e01e096-ae00-11e4-86ad-64c502205f8a.png)

**Comment by RomainBSQT** [comment]
Hello Hayrer,
Did you find a solution to your problem ? Im encountering the exact same behaviour.

**Comment by hayrer** [comment]
Yes, i changed it to this library https://github.com/jgh-/VideoCore, it is much faster and optimized, and it is open source.

**Comment by RomainBSQT** [comment]
Ok, i'll look into it !
Thanks

**Comment by iremk** [comment]
Hi , I am facing the same issue here, did you find a solution for that? When I try to toggle camera it just crashes.

**Comment by shravanteegala** [comment]
Hi, 
Toggle Camera button is disable mode,but no video capture how it is handle the issue ?
